test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -108,7 +108,6 @@ public static void setup(@NonNull Context context) {
         intent.setComponent(new ComponentName("com.linkedin.android.testbutler",
                 "com.linkedin.android.testbutler.ButlerService"));
 
-        context.startService(intent);
```

Comment:
@kkoser can you test this and make sure it still works for emulators with older API levels?

Reply:
I tried running the demo on an API 16 emulator. Service binding worked fine -- 4 of 8 passed, and the 4 that failed were due to features that don't exist / aren't supported in API 16.

Reply:
I  was also able to verify this on api22 that the service was still started